docs/reference-manual/processes/library/lfo.md

```diff
@@ -9,3 +9,28 @@ grand_parent: Reference
 
 permalink: /processes/lfo.html
 ---
+# LFO
+
+![LFO]({{ site.img }}/reference/processes/lfo.png "LFO Example")
+
+You can use the Low Frequency Oscillator in order to modify a sound.
```

Comment:
not really, it generates a value which can be used for controlling any other parameter